src/components/ApplicationGroup/AppGroupAppFilter.tsx

```diff
@@ -87,11 +88,14 @@ export default function AppGroupAppFilter() {
         if (selectedFilterTab === AppFilterTabs.APP_FILTER) {
             setSelectedAppList(selectedValue)
             setSelectedGroupFilter([])
+            setFilterInLocalStorage(filterParentType, resourceId, selectedValue, [])
```

Comment:
NOTE: Please discuss this before moving ahead with this comment

Ideally this should be done through useStateFilters or there should be a mechanism to auto sync state with local storage, after discussion can take call.